src/clients/rust/build.rs

```diff
@@ -1,5 +1,14 @@
-use ignore::Walk;
 use std::{env, fs, path::Path};
```

Comment:
Whoa, the permissions change to 100755 looks suspicious! We probably want to keep as 644 right? Funny thing is that we already have one accidentally executable in jni.zig!